nx-X11/programs/Xserver/hw/nxagent/Keyboard.c

```diff
@@ -1863,10 +1863,9 @@ void nxagentKeycodeConversionSetup(void)
     fprintf(stderr, "Failed to create the keyboard file\n");
   }
 
-  if (nxagentOption(ClientOs) == ClientOsLinux &&
-            drules != NULL && dmodel != NULL &&
-                (strcmp(drules, "evdev") == 0 ||
-                    strcmp(dmodel, "evdev") == 0))
+  if (drules != NULL && dmodel != NULL &&
```

Comment:
Even worse, while the server machine might have `rules` and `model` defined to `evdev` (because we currently really only support Linux as a server OS only in the first place), this change will also enable keycode conversion for OS X and Windows **clients** (i.e., the remote counterpart) which differs from the old behavior. That may or may not be a good thing...